proto/src/determined/job/v1/job.proto

```diff
@@ -175,9 +179,20 @@ message QueueControl {
     int32 priority = 5;
     // The desired job weight in fairshare scheduler.
     float weight = 6;
+    // Name of the target resource_manager & resource_pool to move the job to.
+    Resources resources = 7;
   }
 }
 
+// Describes a resource manager and resource pool, for use in multi-Resource
+// Manager environments.
+message Resources {
```

Comment:
i like ResourcePoolQualifier, if i see Resources, I'm not sure i'd expect what it contains (bit general)